recipe/meta.yaml

```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
unfortunately i don't think the bot knows how to change build numbers, so if you do want to add these, it's probably easiest to put this back to 0 and use a new date for the version (2021.09.09)